shared/bash_remediation_functions/package_install.sh

```diff
@@ -0,0 +1,36 @@
+# Function to install packages on RHEL, Fedora, Debian, and possibly other systems.
+#
+# Example Call(s):
+#
+#     package_install aide
+#
+function package_install {
+
+# Load function arguments into local variables
+local package="$1"
+
+# Check sanity of the input
+if [ $# -ne "1" ]
+then
+  echo "Usage: package_install 'package_name'"
+  echo "Aborting."
+  exit 1
+fi
+
+if which dnf ; then
+  if ! rpm -q --quiet "$package"; then
+    dnf install -y "$package"
+  fi
+elif which yum ; then
+  if ! rpm -q --quiet "$package"; then
```

Comment:
I think it is possible to have `yum` on Debian.
https://packages.debian.org/jessie/yum
Will this work in that case?

Reply:
it should still work

Reply:
YUM would be on debian, but not the RPM command.

Reply:
`rpm` is listed as dependency of `yum`, so both will be there even on debian.